SIS_import/sis_set_url.rb

```diff
@@ -32,6 +32,13 @@
 @esbSecret=""
 @esbUrl=""
 @esbTokenUrl=""
+# ESB token and valid time
+@esbToken=nil
+@esbTokenStartTime=""
```

Comment:
Why set this to an empty string?  Why not nil?  Or better yet, since it will contain a number later, use a number that couldn't possibly be a valid time, like zero or negative one.